dicoogle/src/main/resources/webapp/js/stores/requestStore.js

```diff
@@ -0,0 +1,28 @@
+import Reflux from "reflux";
+
+import { RequestActions } from "../actions/requestActions";
+
+const RequestStore = Reflux.createStore({
+  listenables: RequestActions,
+
+  init: function() {
+    this._contents = "";
+  },
+
+  onQuery: function(data) {
+    this._contents = data
+    self.trigger({
+      contents: this._contents
+    });
+  },
+
+  get: function() {
+    return {
+      contents: this._contents
+    };
+  }
+});
+
+export { RequestStore };
+
+window.store = RequestStore;
```

Comment:
Is this line needed?

Reply:
Not needed, and neither was the trigger above. Removed both.